js/adapt-devtools.js

```diff
@@ -278,6 +280,48 @@ define([
 			Adapt.trigger('drawer:closeDrawer');
 		},
 
+		/*************************************************/
+		/***************** COMPLETE MENU *****************/
+		/*************************************************/
+
+		_checkCompleteMenuVisibility:function() {
+			var currentModel = Adapt.findById(Adapt.location._currentId);
+
+			if (currentModel.get('_type') != 'menu' && currentModel.get('_type') !== "course" ) {
+				this.$('.complete-menu').addClass('display-none');
+				return;
+			}
+
+			var cond = currentModel.has('_isInteractionsComplete') ? {'_isInteractionsComplete':false} : {'_isInteractionComplete':false};
```

Comment:
This check can be dropped now that the framework dependency is >= 2.2: only `isInteractionComplete` is relevant